src/jsMain/js/simulator/windows/ShaderEditorWindow/ShaderEditorWindow.jsx

```diff
@@ -89,6 +97,21 @@ const ShowEditorWindow = (props) => {
     console.log(`previewShow!`);
   };
 
+  const handleTabChange = (event, newValue) => {
+    let src = openShaders[newValue].src;
```

Comment:
Ultra nitpick, doesn't actually need to change for this pull request:

this could be 
```
const src = openShaders[newValue].src;
```
beause we never reassign src in the scope of this function.


Total nitpick, does not absolutely need to change at all...

Reply:
To add onto this nitpick

we could use object de-structuring here so:
```
const { src } = openShaders[newValue];
```
is the exact same thing as
```
const src = openShaders[newValue].src;
```